docker-compose/kafka.yaml

```diff
@@ -89,7 +89,7 @@ services:
     environment:
       CONFIG_FILEPATH: /tmp/config.yml
     volumes:
-      - ./config/redpanda.yaml:/tmp/config.yml
+      - ./config/redpanda-admin.yaml:/tmp/config.yml
```

Comment:
shouldn't we use the security model as default. We could add another redpanda service with a unique profile that is called: redpanda-admin

Reply:
For local development I think you should have admin rights in redpanda as default. The only reason why we include the cluster config (IMO) is to be able to understand/debug why you are unable to do certain things on a more locked down cluster

Reply:
I agree to a point, but opensearch and kafka is behind OAuth2 - so it is a bit inconsistent/confusing in my mind. However, the developers do not actively connect and interact with redpanda, like they do with the core services. We can let it be as-is i guess.